### PR TITLE
Feature/better trade ui (redo correclty)

### PR DESCRIPTION
**Pull request by sboigelot**
A redo of #1068 branching correclty from upstream/master and not my fork/master

Very small PR to align the trade interface in a grid and make it readable

image

### CODE REVIEW

File: Assets/Scripts/UI/DialogBox/Trade/DialogBoxTrade.cs [hunk 1]
```diff
@@ -126,9 +126,9 @@ private void BuildInterface()
 
     private void BuildInterfaceHeader()
     {
-        float tradeAmount = trade.TradeCurrencyBalanceForPlayer;
+        float tradeAmount =  trade.TradeCurrencyBalanceForPlayer;
         PlayerCurrencyBalanceText.text = string.Format("{0} {1}", Math.Round(trade.Player.Currency.Balance + tradeAmount, 2), trade.Player.Currency.ShortName);
         TraderCurrencyBalanceText.text = string.Format("{0} {1}", Math.Round(trade.Trader.Currency.Balance - tradeAmount, 2), trade.Trader.Currency.ShortName);
-        TradeCurrencyBalanceText.text = tradeAmount.ToString();
+        TradeCurrencyBalanceText.text = Math.Round(tradeAmount, 2).ToString();
```
Comment (Grenkin1988) [inline Assets/Scripts/UI/DialogBox/Trade/DialogBoxTrade.cs]:
```
// just two decimal places
string.Format("{0:0.##}", 123.4567);      // "123.46"
string.Format("{0:0.##}", 123.4);         // "123.4"
string.Format("{0:0.##}", 123.0);         // "123"
```

Comment (Mizipzor) [inline Assets/Scripts/UI/DialogBox/Trade/DialogBoxTrade.cs]: Yes, here `string.Format` is the thing to use.

Comment (sboigelot) [inline Assets/Scripts/UI/DialogBox/Trade/DialogBoxTrade.cs]: Good comment, I'll use {0:N2} for correct currency format without symbol

**Comment by Mizipzor** [comment]
Looks good. Is there a good way to instantly spawn a trader? Didnt actually test this new gui, I was more concerned that it doesn break anything current, and it doesnt as far as i can tell.

**Comment by sboigelot** [comment]
@Mizipzor yes there is a simple way to spawn a trader, it's a temporary contextual menu on the landing pad: merged in #817

ezgif com-optimize 1

This is just a very small adjustment on the UI layout. Thanks for the merge!

**Pull request merged**